recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -24,9 +24,30 @@ ANDROID_VARIANT ?= "eng"
 SOC_FAMILY ?= "default"
 
 ANDROID_OUT_DIR_COMMON_BASE = ""
-DDK_KM_PREBUILT_MODULE ?= ""
-TARGET_PREBUILT_KERNEL ?= ""
-DDK_UM_PREBUILDS ?= ""
+
+python __anonymous () {
+    def create_param_str():
+         import os
+         path_not_exists_msg = "The %s does not exists."
+         d.setVar(pname,'')
+         value = d.getVar(vname);
+         if value != None:
+            if not os.path.exists(value):
+                  bb.fatal(path_not_exists_msg % value)
+            d.setVar(pname, "%s=%s" % (vname, value))
+
+    pname='PARAM_DDK_KM_PREBUILDS_MODULE'
+    vname='DDK_KM_PREBUILDS_MODULE'
+    create_param_str()
```

Comment:
This is a quite unpythonic way to pass parameters. 

Why not `create_param_str('PARAM_DDK_KM_PREBUILDS_MODULE', 'DDK_KM_PREBUILDS_MODULE')`?

Reply:
i ma not an expert in python, but , the yocto version brought me an error  like 'unparsed line' when i used the parameters in the function